scipy/optimize/_linprog.py

```diff
@@ -470,11 +473,8 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
      success: True
            x: array([ 9.99999989, -2.99999999]) # may vary
 
-    Note that the default method for `linprog` is 'interior-point', which is
-    approximate by nature. If you need greater accuracy, try 'revised simplex'.
-
-    >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds],
-                      method = 'revised simplex')
+    If you need greater accuracy, try 'revised simplex'.
+    >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds], method='revised simplex')
```

Comment:
I think you also need a blank line here.

Reply:
Thanks!